.github/workflows/kotlin_ci.yml

```diff
@@ -11,18 +11,32 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v3
-    - name: set up JDK 17
+    - name: Checkout target branch
+      uses: actions/checkout@v3
+      with:
+        ref: main
+
+    - name: Fetch all branches
+      run: git fetch origin
+
+    - name: Merge head branch into target branch
+      run: git merge origin/${{ github.event.pull_request.head.ref }}
```

Comment:
Why do you `fetch` all branches first and the `merge`. Can't you just use the command `pull`?

Reply:
done